docs/release-notes/4.1.4.md

```diff
@@ -0,0 +1,41 @@
+# Release notes for Gluster 4.1.4
+
+This is a bugfix release. The release notes for [4.1.0](4.1.0.md),
+ [4.1.1](4.1.1.md), [4.1.2](4.1.2.md) and [4.1.3](4.1.3.md) contains a
+listing of all the new features that were added and bugs fixed in the
+GlusterFS 4.1 stable release.
+
+## Major changes, features and limitations addressed in this release
+
+This release contains a fix for a security vulnerability in Gluster as follows,
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10907
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10904
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10911
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10913
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10923
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10930
+
+Plus to resolve one of the security vulnerability following limitations were made
```

Comment:
To resolve, one of the security vulnerabilities

same comment for the other file